main.c

```diff
@@ -0,0 +1,194 @@
+#include <ctype.h>
+#include <math.h>
+#include <stdio.h>
+#include <stdlib.h>
+#define N 50
+
+int name_verification(char *input_arr) {
+  char name_of_figure[6] = "Circle";
```

Comment:
https://csc-software-development.readthedocs.io/ru/2023/code-style.html#magic-numbers

должно быть:
```suggestion
  char name_of_figure[6] = "circle";
```